openapi-docs/README.md

```diff
@@ -0,0 +1,133 @@
+
+To use the library locally without publishing to a remote npm registry, first install the dependencies by changing into the directory containing `package.json` (and this README). Let's call this `JAVASCRIPT_CLIENT_DIR`. Then run:
+
+```shell
+npm install
+```
+
+Next, [link](https://docs.npmjs.com/cli/link) it globally in npm with the following, also from `JAVASCRIPT_CLIENT_DIR`:
+
+```shell
+npm link
+```
+
+To use the link you just defined in your project, switch to the directory you want to use your ambianic_serverless_functions_api_definition from, and run:
+
+```shell
+npm link /path/to/<JAVASCRIPT_CLIENT_DIR>
+```
+
+Finally, you need to build the module:
+
+```shell
+npm run build
+```
+
+#### git
+
+If the library is hosted at a git repository, e.g.https://github.com/GIT_USER_ID/GIT_REPO_ID
+then install it via:
+
+```shell
+    npm install GIT_USER_ID/GIT_REPO_ID --save
+```
+
+### For browser
+
+The library also works in the browser environment via npm and [browserify](http://browserify.org/). After following
+the above steps with Node.js and installing browserify with `npm install -g browserify`,
+perform the following (assuming *main.js* is your entry file):
+
+```shell
+browserify main.js > bundle.js
+```
+
+Then include *bundle.js* in the HTML pages.
+
+### Webpack Configuration
+
+Using Webpack you may encounter the following error: "Module not found: Error:
+Cannot resolve module", most certainly you should disable AMD loader. Add/merge
+the following section to your webpack config:
+
+```javascript
+module: {
+  rules: [
+    {
+      parser: {
+        amd: false
+      }
+    }
+  ]
+}
+```
+
+## Getting Started
+
+Please follow the [installation](#installation) instruction and execute the following JS code:
+
+```javascript
+var AmbianicServerlessFunctionsApiDefinition = require('ambianic_serverless_functions_api_definition');
+
+
+var api = new AmbianicServerlessFunctionsApiDefinition.DefaultApi()
+var callback = function(error, data, response) {
+  if (error) {
+    console.error(error);
+  } else {
+    console.log('API called successfully.');
+  }
+};
+api.netlifyFunctionsCustomersGet(callback);
+
+```
+
+## Documentation for API Endpoints
+
+All URIs are relative to *https://ambianic-functions.netlify.app*
+
+Class | Method | HTTP request | Description
+------------ | ------------- | ------------- | -------------
+*AmbianicServerlessFunctionsApiDefinition.DefaultApi* | [**netlifyFunctionsCustomersGet**](docs/DefaultApi.md#netlifyFunctionsCustomersGet) | **GET** /.netlify/functions/customers | 
+*AmbianicServerlessFunctionsApiDefinition.DefaultApi* | [**subscriberPost**](docs/DefaultApi.md#subscriberPost) | **POST** /subscriber | 
+*AmbianicServerlessFunctionsApiDefinition.DefaultApi* | [**subscriptionDelete**](docs/DefaultApi.md#subscriptionDelete) | **DELETE** /subscription | 
+
+
+## Documentation for Models
+
+
+
+## Documentation for Authorization
+
+All endpoints do not require authorization.
```

Comment:
This cannot be a true statement.

Reply:
It is.
It refers to if any form of authorization is needed to make a request to this endpoints.

The endpoints are currently public.

Reply:
@vickywane How are we going to protect these public APIs from malicious attacks that result into random notifications sent to users?

Reply:
@vickywane this comment is still unresolved. What protection do we have against malicious use of these public APIs?